src/test/java/com/microsoft/sqlserver/jdbc/fips/FipsTest.java

```diff
@@ -86,7 +86,7 @@ public void fipsPropertyTest() throws Exception {
         props.remove(Constants.ENCRYPT);
         props.remove(Constants.TRUST_SERVER_CERTIFICATE);
 
-        try (Connection con = PrepUtil.getConnection(connectionString, props)) {
+        try (Connection con = PrepUtil.getConnection(connectionString + "encrypt=false", props)) {
```

Comment:
```suggestion
        try (Connection con = PrepUtil.getConnection(connectionString + ";encrypt=false", props)) {
```

Reply:
no, not trying to change connectionString, just AEConnectionString

Reply:
Not needed.

Reply:
hmm this shouldn't be here it's being ignored, should be removed